CareKitUI/CareKitUI/iOS/Task/SliderTaskView.swift

```diff
@@ -0,0 +1,243 @@
+    /// - Parameter sliderStyle: The style of the slider, either the SwiftUI system slider or the custom filler slider.
+    /// - Parameter action: Action to perform when the button is tapped.
+    /// - Parameter header: Header to inject at the top of the card. Specified content will be stacked vertically.
+    init(instructions: Text? = nil, isComplete: Bool,
+         initialValue: CGFloat? = nil, value: Binding<CGFloat>, range: ClosedRange<CGFloat>, step: CGFloat = 1,
+         minimumImage: Image? = nil, maximumImage: Image? = nil, sliderStyle: OCKSliderStyle, isSliderTop: Bool = false,
+         action: @escaping (Double) -> Void,
+         @ViewBuilder header: () -> Header) {
+        self.init(isHeaderPadded: false, isSliderViewPadded: true, instructions: instructions, header: header, sliderView: {
+            _SliderTaskViewSliderView(initialValue: initialValue,
+                                  value: value,
+                                  range: range,
+                                  step: step,
+                                  isComplete: isComplete,
+                                  minimumImage: minimumImage,
+                                  maximumImage: maximumImage,
+                                  sliderStyle: sliderStyle,
+                                  isSliderTop: isSliderTop,
+                                  action: action)
+        })
+    }
+}
+
+public extension SliderTaskView where Header == _SliderTaskViewHeader, SliderView == _SliderTaskViewSliderView {
+    
+    /// Create an instance.
+    /// - Parameter title: Title text to display in the header.
+    /// - Parameter detail: Detail text to display in the header.
+    /// - Parameter instructions: Instructions text to display under the header.
+    /// - Parameter isComplete; True if the button under the slider is in the completed.
+    /// - Parameter initialValue: The initial value the slider begins on. Default value is the midpoint of the range.
+    /// - Parameter value: The binded value that the slider will reflect
+    /// - Parameter range: The range that includes all possible values.
+    /// - Parameter step: Value of the increment that the slider takes. Default value is 1
+    /// - Parameter minimumImage: Image to display to the left of the slider. Default value is nil.
+    /// - Parameter maximumImage: Image to display to the right of the slider. Default value is nil.
+    /// - Parameter sliderStyle: The style of the slider, either the SwiftUI system slider or the custom filler slider.
+    /// - Parameter action: Action to perform when the button is tapped.
+    init(title: Text, detail: Text? = nil, instructions: Text? = nil, isComplete: Bool,
+         initialValue: CGFloat? = nil, value: Binding<CGFloat>, range: ClosedRange<CGFloat>, step: CGFloat = 1,
+         minimumImage: Image? = nil, maximumImage: Image? = nil, sliderStyle: OCKSliderStyle, isSliderTop: Bool = false,
+         action: @escaping (Double) -> Void) {
```

Comment:
The type of the `value` accessible in the action should match the type of the `value` passed in during init.